keepassxc-browser/background/page.js

```diff
@@ -11,7 +11,7 @@ const defaultSettings = {
     showNotifications: true,
     showLoginNotifications: true,
     saveDomainOnly: true,
-    automaticReconnect: true,
+    automaticReconnect: false,
```

Comment:
Recommend we rename the setting entirely so that people who had it enabled previously will not keep it enabled. Recommend ```autoReconnect```.